NETWORK/NetworkConcealPlayer.md

```diff
@@ -15,3 +15,42 @@ This is what R* uses to hide players in MP interiors.
 * **toggle**: 
 * **p2**: 
 
+To manage player visibility with NetworkConcealPlayer, here’s a solid approach:
+
+**General Population (players not in any instance):**
+
+* Use NetworkConcealPlayer to hide players who are in any instance. This way, general population players won’t see or interact with instance players.
+
+**Instance Players (players in a specific instance):**
+
+* Use NetworkConcealPlayer to hide players who aren’t in the same instance. Instance players can still see and interact with the general population but not with players in other instances.
+
+This setup keeps instance players separate from each other while allowing interaction with the general population.
+
+
+```lua
+-- Function to manage player visibility
+function concealPlayers(instanceId)
+    local allPlayers = GetPlayers()
+
+    for _, player in ipairs(allPlayers) do
+        local playerInstance = GetPlayerInstance(player)  -- You need to define this function
```

Comment:
You don't need an instance, not sure why this method exists.
All you need is a valid player index and that player needs to be networked, also known as a networked object or `netObj`.

Reply:
Yeah, I'm aware it's not required, I was just showcasing a use case scenario, for interiors (like GTA:O apartments)

Reply:
It was to provide an efficient scenario, with interior instancing for people wanting to use the general population (which to my knowledge, is not supported by routing bucket?)

Reply:
Hmm, I still don't think it's a good idea to put undefined functions in the documentation.

Reply:
That's very true, but it's very hard to provide an example without some kind of pseudo code

Reply:
It might be better to use a state bag in this case, or define the function with a hard coded instance id and let the end user know this is where they should add how they do their instances

Something like this might also work

```lua
function GetPlayerInstance(player)
    local playerServerId = GetPlayerServerId(player)
    return Player(playerServerId).state.instance_id or 0
end
```

You also don't need the extra if/else branch, you can just check if the instance ids match and if they don't just conceal them.

Reply:
The else was added in case that people left the "instance" they would have to be removed from the NET_CONCEAL filter in R*

State bags are also correct, yes, I just wrote what I assumed to be the best example for a simple use-case scenario